### ISSUE TITLE
Accidentally marked my project wrong, and now it won't load

### ISSUE THREAD

**Issue by ghost** [tom-englert/ProjectConfigurationManager#21]
Preface - I'm a newbie to Visual Studio, C#, and advanced programming in general, so it's entirely possible that this is my own fault, or that an obvious, simple solution exists. 
That said, here's what's going on:

**Background:**
I'm writing a Windows Forms program - my objective in installing ProjectConfigurationManager was to use Fody to embed a DLL into the EXE to make distribution easy (that's how that works, right?).
ProjectConfigurationManager's intended role was to show what Fody was up to, and that's all I wanted from it.

**Issue:**
I accidentally clicked the checkbox for 'Database' under the 'Project Types' tab - a simple misclick.
Now I can't uncheck it, I can't check anything else, and VS will no longer load the project because 'This project is incompatible with the current version of Visual Studio' - which it was written in.
I unloaded the project in ProjectConfigurationManager in the hopes that the changes were not saved (they were).
Nothing else was changed.
What have I gone and done? 
How do I undo it? 
Which files does this setting modify, and can I edit them back to working order manually?

I appreciate the (free!) great effort put in to maintain a project like this, and I hate to ask for help, but please help.

TL;DR -  The core issue here is the inability to undo a misclick in the under the 'Project Types' tab through the GUI

**Comment by ghost** [comment]
Fixed my specific version of the issue - I manually edited the .csproj and replaced the ProgramType GUID with one from a fresh, correctly marked project.

The base issue with ProjectConfigurationManager still stands - it should not be possible to mark a project in such a way as to be impossible to undo.

**Comment by tom-englert** [comment]
I just have tried to reproduce this - as expected PCM automatically undoes the changes if a project can't be loaded again after a change. The bug is in Visual Studio, that it does not let you reload the project unless you close and reopen the solution. After doing so I could load the project again.

So the undo here is automatically done!

A manual undo can be found in every SCM, so I saw no need to repeat this function here.
